docker/README.md

```diff
@@ -1,23 +1,41 @@
+# Introduction
+
+This README describes two ways to deploy the containers for web UI and the workers.
+
+The first one describes how to deploy an openQA environment using docker with
+the Fedora images or images built locally.
+
+The second one uses a docker-compose to deploy a complete web UI with HA for the
```

Comment:
```suggestion
The second one uses docker-compose to deploy a complete web UI with HA for the
```

Reply:
I think we should do that afterwards.

Reply:
Created https://github.com/os-autoinst/openQA/pull/3507